smoke/IotEdgeQuickstart/Quickstart.cs

```diff
@@ -12,6 +12,7 @@ public class Quickstart : Details.Details
     {
         readonly LeaveRunning leaveRunning;
         readonly bool noDeployment;
+        private readonly bool noVerifyScenarios;
```

Comment:
`private` is the default. I assume R# will complain about this?

Reply:
R# didn't complain, but Roslly did. Fixed. 

---
In reply to: [208328608](https://github.com/Azure/iotedge/pull/131#discussion_r208328608) [](ancestors = 208328608)